tools/ci_build/github/azure-pipelines/templates/c-api-cpu.yml

```diff
@@ -500,11 +500,12 @@ stages:
       displayName: 'Clean Agent Directories'
       condition: always()
 
-- stage: Nodejs_Packaging_CPU
+- stage: Nodejs_Packaging
   dependsOn:
   - Linux_C_API_Packaging_CPU
   - MacOS_C_API_Package_Publish
   - Windows_Packaging_CPU_x64_${{ parameters.BuildVariant }}
```

Comment:
I think `Windows_Packaging_CPU_x64_${{ parameters.BuildVariant }}` is no longer need?